packages/opine/opine.0.1.3/opam

```diff
@@ -0,0 +1,52 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Python AST unparse implementation in OCaml"
+description:
+  "Opine is an OCaml library that unparses the python AST back to python code."
```

Comment:
The description talks about an OCaml library, however looking at the code, the `opine` library is not installed. Could you add the missing `(public_name opine)` ?

Reply:
Yes, I did. Didn't realize that's how libraries were installed via dune/opam. I tested in my local machine by running ```bash opam install . ``` and importing ```Opine``` as a library. So, I'm hoping that this will work!